src/location-registry/helpers/helper.py

```diff
@@ -9,63 +9,30 @@
 from shapely.ops import transform
```

Comment:
Could we use a more descriptive name for this file? Since it is already under helpers, we could take advantage of that fact and give it a more descriptive name?